src/pages/Defi/views/StakingVaults.tsx

```diff
@@ -12,9 +16,50 @@ const DefiHeader = () => {
   )
 }
 
+const FoxFarmCTA = () => {
+  const ethLogoSrc = 'https://assets.coincap.io/assets/icons/256/eth.png'
```

Comment:
We should use smaller icon, grep for `2x.png` in the codebase or have a look in `src/test/constants`
```suggestion
  const ethLogoSrc = 'https://assets.coincap.io/assets/icons/eth@2x.png'
```

Reply:
Oh, nice to learn, I searched something like that when reviewing